indico/MaKaC/PDFinterface/conference.py

```diff
@@ -609,30 +604,24 @@ def sortEntries(x,y):
     else:
         return cmp(x.getTitle(), y.getTitle())
 
-class TimeTablePlain(PDFWithTOC):
 
-    def __init__(self,conf,aw,showSessions=[],showDays=[],sortingCrit=None, ttPDFFormat=None,
-                 pagesize='A4', fontsize = 'normal', firstPageNumber = 1, showSpeakerAffiliation=False, tz=None):
-        self._conf=conf
-        if not tz:
-            self._tz = self._conf.getTimezone()
-        else:
-            self._tz = tz
-        self._aw=aw
-        self._tz = DisplayTZ(self._aw,self._conf).getDisplayTZ()
-        self._showSessions=showSessions
-        self._showDays=showDays
+class TimeTablePlain(PDFWithTOC):
+    def __init__(self, conf, aw, showSessions=None, showDays=None, sortingCrit=None, ttPDFFormat=None,
+                 pagesize='A4', fontsize='normal', firstPageNumber=1, showSpeakerAffiliation=False, tz=None):
+        self._conf = conf
+        self._aw = aw
+        self._tz = DisplayTZ(self._aw, self._conf).getDisplayTZ()
+        self._showSessions = showSessions
+        self._showDays = showDays
         self._ttPDFFormat = ttPDFFormat or TimetablePDFFormat()
-        story=[]
-        if self._ttPDFFormat.showCoverPage():
-            story=None
-        PDFWithTOC.__init__(self, story=story, pagesize = pagesize, fontsize=fontsize, firstPageNumber = firstPageNumber)
-        self._title= _("Programme")
-        self._doc.leftMargin=1*cm
-        self._doc.rightMargin=1*cm
-        self._doc.topMargin=1.5*cm
-        self._doc.bottomMargin=1*cm
-        self._sortingCrit=sortingCrit
+        story = None if self._ttPDFFormat.showCoverPage() else []
+        PDFWithTOC.__init__(self, story=story, pagesize=pagesize, fontsize=fontsize, firstPageNumber=firstPageNumber)
+        self._title = _("Programme")
+        self._doc.leftMargin = 1 * cm
+        self._doc.rightMargin = 1 * cm
+        self._doc.topMargin = 1.5 * cm
+        self._doc.bottomMargin = 1 * cm
```

Comment:
I would rename `cm` to `1cm` and avoid all those silly `1 * cm` (I know it's legacy code!)